csharp-mode-tests.el

```diff
@@ -108,20 +118,21 @@
    "var import = true;"
    "import" 'font-lock-variable-name-face))
 
-(ert-deftest fontification-of-literals-allows-multi-line-strings ()
-  (require 'assess)
-  (should (assess-face-at=
-           "string Literal = \"multi-line\nstring\";"
-           'csharp-mode
-           ;; should be interpreted as error
-           18 'font-lock-warning-face
-           ))
-  (should (assess-face-at=
-           "string Literal = @\"multi-line\nstring\";"
-           'csharp-mode
-           ;; should not be interpreted as error because of @
-           19 'font-lock-string-face
-           )))
+;; TODO: Should we really behave like this? The new CC Mode multiline strings doesn't
```

Comment:
If the compiler errors on it, it makes sense for us to error on it too, at least in theory.

On the flip side, most people aren't going to write code like this, and if it's hard to do 100% right, let's make sure the things we do right is the things people will need/want to be right :smile:

Reply:
Yes, It does make sense to error on it. However, I'm not sure how hard this behavior is to get right anymore.

However, I will ask Alan about this :)